src/Breadcrumb.js

```diff
@@ -0,0 +1,47 @@
+import React, { cloneElement } from 'react';
+import classNames from 'classnames';
+import BootstrapMixin from './BootstrapMixin';
+import ValidComponentChildren from './utils/ValidComponentChildren';
+
+const Breadcrumb = React.createClass({
+  mixins: [BootstrapMixin],
+
+  propTypes: {
+    /**
+     * CSS classes for the wrapper `ol` element
+     */
+    className: React.PropTypes.string,
+    /**
+     * HTML id for the wrapper `ol` element
+     */
+    id: React.PropTypes.string
```

Comment:
What's the point in specifying this as a propType though? You're not using it in the component code, nor attaching any special semantics to it - it's just getting passed through as an HTML attribute. Better to just drop this propType entirely.